pkg/network/tracer/tracer.go

```diff
@@ -265,6 +250,24 @@ func NewTracer(config *config.Config) (*Tracer, error) {
 	return tr, nil
 }
 
+func newReverseDNS(cfg *config.Config, m *manager.Manager, pre410Kernel bool) (network.ReverseDNS, error) {
+	if !cfg.DNSInspection {
+		return network.NewNullReverseDNS(), nil
+	}
+
+	if pre410Kernel {
+		log.Warn("dns inspection not supported by this kernel version. please refer to our documention.")
```

Comment:
Can we be more specific about the kernel versions supported? Also, I think an explicit link to the documentation is helpful.

Reply:
Honestly I don't know how we should we phrase this. Could you give a suggestion?
* I believe socket filter program types were introduced in Kernel 3.19 (https://elixir.bootlin.com/linux/v3.19/source/include/uapi/linux/bpf.h#L120)
* However system-probe requires Kernel 4.4.0+ with the exception of the RHEL/CentOS kernels that have eBPF stuff backported (3.10)
* However CentOS7 and RHEL8 don't support socket filter programs 😂 ;
So based on this rather confusing criteria I tried to avoid specifying a kernel version, but I'm open to change it if there's a better way.

PS.: I also don't know what's the context for using 4.1.0 in these validations.

Reply:
I also don't understand why we check for 4.1.0 and not 4.4.0? It would be good for us to understand that and document it...somewhere. Not necessary for this PR though.

I agree that it would be better if we did feature detection instead of checking for specific kernel versions. That is probably a different PR though.

For now can we log
`DNS inspection not supported by kernel versions < 4.1.0. Please see https://docs.datadoghq.com/network_performance_monitoring/installation for more details.`